src/index.json

```diff
@@ -2181,6 +2181,49 @@
                 "sha256Digest": "42c6500e5587184026a398147b0a7b1a9cb7e59642de9b932dfcbe1a90f14697"
             }
         ],
+        "vmware-cs": [
+            {
+                "downloadUrl": "https://github.com/Azure/az-vmware-cli/releases/download/v0.1.0/vmware_cs-0.1.0-py2.py3-none-any.whl",
+                "filename": "vmware_cs-0.1.0-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.isPreview": true,
+                    "azext.maxCliCoreVersion": "2.1.0",
+                    "azext.minCliCoreVersion": "2.0.67",
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "shimitta@microsoft.com",
+                                    "name": "Microsoft Corporation",
+                                    "role": "author"
+                                }
+                            ],
+                            "document_names": {
+                                "description": "DESCRIPTION.rst"
+                            },
+                            "project_urls": {
+                                "Home": "https://github.com/Azure/az-vmware-cli"
+                            }
+                        }
+                    },
+                    "extras": [],
+                    "generator": "bdist_wheel (0.30.0)",
+                    "license": "MIT",
+                    "metadata_version": "2.0",
+                    "name": "vmware-cs",
+                    "run_requires": [
```

Comment:
remove run_requires entirely
do the same in setup.py.
and you will have to rebuild and upload again the wheel

Reply:
Done.